scanners/wpscan/README.md

```diff
@@ -3,7 +3,7 @@ title: 'WPScan'
 category: 'scanner'
 type: "CMS"
 state: "released"
-appVersion: "3.8.17"
+appVersion: "v3.8.19"
```

Comment:
I am not sure if the changes in the README file have been necessary because the files should be updated automatically via Helm Docs, but they are fine of course.